src/sync/lookupApiUtils.js

```diff
@@ -37,20 +45,22 @@ export const createPolicyRecord = policy => {
   UIDatabase.write(() => createRecord(UIDatabase, 'InsurancePolicy', policyRecord));
 };
 
-const getQueryString = params =>
-  params.reduce((queryString, param) => {
-    const [[key, value]] = Object.entries(param);
-    if (!value) return queryString;
-    const paramString = `${key}=${SEPARATORS.QUERY_WILDCARD}${value}${SEPARATORS.QUERY_WILDCARD}`;
-    const paramSeparator = queryString.length > 0 ? SEPARATORS.PARAMETERS : SEPARATORS.QUERY_STRING;
-    return queryString + paramSeparator + paramString;
-  }, '');
+const getQueryString = params => {
+  const object = params.reduce((queryObject, param) => {
+    const [[key, value], [, type]] = Object.entries(param);
+    if (!value) return queryObject;
+    const paramValue = type !== TYPES.DATE ? `@${value}@` : moment(value).format('YYYYMMDD');
```

Comment:
The date format should be `DDMMYYYY`

<img width="393" alt="Screen Shot 2020-05-11 at 1 23 06 PM" src="https://user-images.githubusercontent.com/9192912/81516203-38beff80-938b-11ea-8f7e-40f583e9959a.png">

Reply:
![oops](https://user-images.githubusercontent.com/43223637/81528459-d678f580-93b0-11ea-9d2b-d938df92bf08.gif)

49ca9ccc1f65540776b674b0eeeb5d392fc27b46.

Reply:
![dobby-finger-snap](https://user-images.githubusercontent.com/9192912/81614275-e8e84300-9433-11ea-90df-3cd18af0d947.gif)

( works a treat! thanks )